### PR TITLE
Complete Helper\Order coverage

### PR DESCRIPTION
**Pull request by danac-gs**
# Description
Complete Helper\Order coverage

Fixes: https://app.asana.com/0/1131034199022718/1155354388549289/f

(Optional: Add a changelog by writing a comment after "#changelog" on the next line)

\#changelog

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: Helper/Order.php [hunk 1]
```diff
@@ -287,7 +287,7 @@ public function fetchTransactionInfo($reference, $storeId = null)
      *
      * @throws \Exception
      */
-    private function setShippingMethod($quote, $transaction)
```
Comment (matt-thomason) [inline Helper/Order.php]: What is the reasoning to set all these `protected`? Just for easier testing convenience?

File: Test/Unit/Helper/TestHelper.php [hunk 2]
```diff
@@ -1,29 +0,0 @@
-<?php
-
-namespace Bolt\Boltpay\Test\Unit\Helper;
-
-use PHPUnit\Framework\TestCase;
-
-class TestHelper extends TestCase
```
Comment (matt-thomason) [inline Test/Unit/Helper/TestHelper.php]: Was nothing using this old `TestHelper.php` before? I don't see any updated `use` statements.

Comment (danac-gs) [reply Test/Unit/Helper/TestHelper.php]: It was used from the same folder, in one one place only so far, SessionTest.php. I moved it one folder up, more logical place to be used in other tests / folders.

File: Helper/Order.php (hunk 1 above)
Comment (danac-gs) [reply Helper/Order.php]: Yep. Some were complicated to test as private.

**Pull request merged**